src/ui/public/kfetch/kfetch.ts

```diff
@@ -61,7 +60,7 @@ export async function kfetch(
         query,
       });
 
-      return fetch(fullUrl, restOptions).then(async res => {
+      return window.fetch(fullUrl, restOptions).then(async res => {
```

Comment:
Make it explicit that we rely on the browser `fetch` (and `whatwg-fetch`, not `node-fetch`)